src/System.Console/pkg/unix/System.Console.pkgproj

```diff
@@ -7,9 +7,9 @@
     <PreventImplementationReference>true</PreventImplementationReference>
   </PropertyGroup>
   <ItemGroup>
-    <ProjectReference Include="..\..\src\System.Console.builds" >
+    <ProjectReference Include="..\..\src\System.Console.csproj" >
       <!-- the binary is not specific to Linux, but we only do a Linux & OSX pass, not Unix-specific pass -->
```

Comment:
Yes we should however I would prefer we not do that entire conversion as part of this PR. The only reason this needed to be touched was because we disabled the filtering.